src/snippets/index.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  httpGETWithURLParams,
+  httpPOSTWithAuthBearer,
+  httpPOSTWithFormSubmit,
+  httpPOSTWithJSONBody,
+  httpPUTWithFileUpload,
+  minimalRequest,
+} from "./http";
+
+console.log(httpGETWithURLParams);
```

Comment:
Remove this please.

Reply:
Of course, thank you for letting me know. Done 😊